share_app/build.gradle

```diff
@@ -24,6 +24,10 @@ android {
 }
 
 dependencies {
+    def lifecycle_version = "2.0.0"
+
+    // ViewModel and LiveData
+    implementation "androidx.lifecycle:lifecycle-extensions:$lifecycle_version"
```

Comment:
this issue can be resolved without using this, any other use case for which you want to use this?

Reply:
I was trying to use ViewModel for this specific issue. What alternate method you think could be implemented? Any hints would be really appreciated.